README.md

```diff
@@ -1,2 +1,50 @@
 # Restaurant-page
-A simple restaurant page,  I used JavaScript alone to generate the entire contents of the website.
+A simple restaurant page,  I used JavaScript alone to generate the entire contents of the website. The scope of this project is a deeper understanding of Webpack and DOM manipulation with JS.
+
+![screenshot](dist/assets/images/screenshot.jpeg)
+![screenshot](dist/assets/images/screenshot2.jpeg)
+
+## Live demo
+
+[live demo](https://mricanho.github.io/Restaurant-page/)
+
+## Built With
+
+- JavaScript
+- HTML
+- Webpack
+- Bulma
+
+## How to start the project from your Local enviroment
+
+- Open your terminal and cd where you want to store the project
+- Run the following command:
+```bash
+https://github.com/mricanho/Restaurant-page.git
+```
+- CD to the directory where you create the clone repository.
+- Run:
+```bash
+npm run build
+```
+- Open the live server on your VScode `Alt + L` then `Alt + o`
```

Comment:
## Improve the README Instructions
- [ ] Some instructions on how to run the project locally seems to be missing here. Do you mind adding them here? That is if we are going with the `start script.`
```suggestion
- Open your terminal, clone the repo, and cd into the cloned repo.
**_Actions:_**
- Run the following command:
```terminal
git clone https://github.com/mricanho/Restaurant-page.git
```
-  change directory to point to the `Restaurant-page` dir.
- Run:
```terminal
npm run build
```

```terminal
npm run start
```

NOTE: Please, these are just but my suggestions. You are free to improve on them in the best way possible.